### ISSUE TITLE
QT Client: a problem with context menu accessibility.

### ISSUE THREAD

**Issue by amirmahdifard** [BearWare/TeamTalk5#1559]
### Description

  in some places, for example, (user accounts list, files list, servers list in connect to a server dialogue), when you press application key to open context menu, NVDA doesn't announce it as expected, and when you press one of your arrow key, it first says (teamtalk5 window) and then it's announcing one of the items in the context menu: it seems that it is stuck and doesn't open the context menu until you press one of your arrow key: it happens not only in context menus only, in some other random place too: for example, if your pressing controle + u in someone's name, the menu will open for you, and you even can change the volume with your arrow keys, but NVDA doesn't announce the numbers, until you press tab and shift tab once to focus on voice volume slider, then if you change the volume with your arrow keys, it will announce the numbers for you: i can tell you that this problem happens with moste dialogues i really like you to fix this problems so NVDA won't stuck anywhere and work normally, i'm not sure if it's QT Accessibility and you have to report to them, or you can fix it your self: if it's QT Accessibility, please report it, or if you can fix it, please do it: QT Client vertion: 5.11.

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [X] Windows
- [ ] macOS
- [ ] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

when you press a button that opens a dialogue like a context menu or other dialogues, the screen reader shouldn't stuck at some point and should focus on the items or if it shouldn't focus on any item, it should at least announce it that you've opened a dialogue

### Actual behavior

when you press a button that opens a dialogue like a context menu or other dialogues, the screen reader is stuck and if it is context menu, it doesn't say Teamtalk5 window as soon as you press the application key, you need to press one of your arrow key to make it out of stuck: and if it's a dialogue that your screen reader should automaticly focus on an item as soon as you've opened it, it doesn't focus and it's stuck, you need to press tab or an arrow key or something to make it focus on something.

### Steps to reproduce problem

1: go to Teamtalk5: 2: go to some place like user accounts list, or files list, or your server list in connect to a server dialogue (because these places are the ones wich i know that this bug happens in): 3: press application key, NVDA says nothing: now, press your up or down arrow key once, NVDA will first say (Teamtalk5 window), then it will announce one of the items: but the strange thing is, that this one doesn't happen with all context menus, just it is happening with these places: for example, if you press the Application key on a channel or on a user, NVDA will say (menu) and then if you press your arrow keys, NVDA will announce the items as expected, so i don't know how they are difrent: another example with not context menu and with a dialogue: 1: in a server, press controle + u on a user to open volume menu for that user, then NVDA will say that the dialogue is opened, but at the end, it doesn't focus on the Voice slider, so if you press your arrow keys, you can change the volume, but NVDA doesn't announce numbers: you have to press tab and shift + tab once to make it focus on the slider: this one also happens with moste dialogues like channel password dialogue, and some other dialogues that you can find out yourself with my explanations.

**Comment by CoBC** [comment]
Already reported to Qt and hopefully fixe in Qt 6.4.1 (15.11.2022) ([QTBUG-105735](https://bugreports.qt.io/browse/QTBUG-105735))

**Comment by amirmahdifard** [comment]
@CoBC but y it doesn't happens in like channels context menu,

**Comment by amirmahdifard** [comment]
@CoBC please fix the other bugs that i've reported, as you know how to fix bugs in QT client and your opening PR for it.
